app/serializers/projects_serializer.rb

```diff
@@ -1,8 +1,4 @@
 class ProjectsSerializer
   include FastJsonapi::ObjectSerializer
   attributes  :mod, :student_projects
-
-  attribute :project_feedback do |object|
```

Comment:
Yep. Turns out this was actually not needed because its on the poro for student projects, the serializer knows it's a collection and pulls it in for that object. It's way smarter than I gave it credit for 😝